### ISSUE TITLE
Update Typegoose dependency 

### ISSUE THREAD

**Issue by alvaaz** [MichalLytek/type-graphql#552]
**Describe the issue**
The example used in this repository for Typegoose use a deprecated package. The correct dependency is [@typegoose/typegoose](https://www.npmjs.com/package/@typegoose/typegoose). When I updated it shows me an error:

**Error output** 

`
[Node] (node:12838) UnhandledPromiseRejectionWarning: Error: Error: Generating schema error
`

**My entity is** 

```
import { prop as Property, getModelForClass } from '@typegoose/typegoose'
import { ObjectId } from 'mongodb'
import { Field, ObjectType } from 'type-graphql'

@ObjectType()
export class Professional {
  @Field()
  readonly _id: ObjectId

  @Field()
  @Property({ required: true })
  firstName: string

  @Field()
  @Property({ required: true })
  lastName: string
}

export const ProfessionalModel = getModelForClass(Professional)
```

**My resolver is**

```
import { Resolver, Mutation, Arg } from 'type-graphql'
import { Professional, ProfessionalModel } from '../entities/Professional'

@InputType()
export class ProfessionalInput implements Partial<Professional> {
  @Field()
  firstName: string

  @Field()
  lastName: string
}

@Resolver()
export class ProfessionalResolver {
  @Mutation(() => Professional)
  async createProfessional(@Arg('options') options: ProfessionalInput): Promise<Professional> {
    try {
      const professional = new ProfessionalModel(options as Professional)

      return await professional.save()
    } catch (err) {
      console.log('Erroooooor', err)
    }
  }
}
```

**Comment by MichalLytek** [comment]
Fixed via 48cc1a9 🔒

**Comment by alvaaz** [comment]
Thanks @MichalLytek. If someone is here because error then typegoose update. 

The problem is that with the new typegoose, you need to add at least one query.
